Library/Source/RuntimeImpl.cpp

```diff
@@ -77,25 +55,27 @@ namespace Babylon
 
     void RuntimeImpl::LoadScript(const std::string& url)
     {
-        auto lock = AcquireTaskLock();
-        auto whenAllTask = arcana::when_all(LoadUrlAsync<std::string>(GetAbsoluteUrl(url).data()), Task);
-        Task = whenAllTask.then(*m_dispatcher, m_cancelSource, [this, url](const std::tuple<std::string, arcana::void_placeholder>& args) {
+        std::scoped_lock lock{ m_taskMutex };
```

Comment:
nit: spacing everywhere

```suggestion
        std::scoped_lock lock{m_taskMutex};
```

Reply:
Did it, though I'm generally not a huge fan of more terse styles. Is there an established convention we're trying to align with on that?

Reply:
LLVM default for clang format